src/services/api/surveyApi.ts

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import { ax } from '../axios';
+import type {
+  SurveyLinkErrorResponse,
+  SurveyLinkRequest,
+  SurveyLinkResponse,
+  SendSurveyLinkRequest,
+  SendSurveyLinkResponse,
+  SendSurveyLinkErrorResponse,
+  SubmitSurveyRequest,
+  SubmitSurveyResponse,
+  SubmitSurveyErrorResponse,
+} from '@/models/survey/surveyApiModels';
+
+// 평가 링크로부터 정보 가져오기
+export const fetchSurveyLink = async (params: SurveyLinkRequest): Promise<SurveyLinkResponse> => {
+  try {
+    const response = await ax.get<SurveyLinkResponse>('/api/v1/peer-reviews/link', {
+      params,
+    });
+    console.log('Fetch Peer Review Link Response:', response.data);
+    return response.data;
+  } catch (error) {
+    if (axios.isAxiosError(error)) {
+      const errorResponse: SurveyLinkErrorResponse = error.response?.data;
+      console.error(`평가 링크 정보 가져오기 실패: ${errorResponse.message || error.message}`);
+      console.error('Full error response:', errorResponse);
+      throw new Error(`평가 링크 정보 가져오기 실패: ${errorResponse.message || error.message}`);
+    } else {
+      console.error(`평가 링크 정보 가져오기 실패: ${error}`);
+      throw new Error(`평가 링크 정보 가져오기 실패: ${error}`);
+    }
+  }
+};
+
+// 평가 링크 발송
+export const sendSurveyLink = async (
+  params: SendSurveyLinkRequest,
+): Promise<SendSurveyLinkResponse> => {
+  try {
+    const response = await ax.post<SendSurveyLinkResponse>(
+      '/api/v1/peer-reviews/link',
+      {},
+      {
+        params,
```

Comment:
여기는 경로에 표시되는 값이니까 다음처럼 쿼리 파라미터 key 값 명시하는게 나중에 안헷갈릴 것 같습니다

```suggestion
        params:  { projectId : params.projectId }
```

Reply:
fetchSurveyLink 도 params 바로 넣어주셨던데, 유지보수 하려면 다 명시하는게 좋지 않을까요?!